src/Microsoft.TestPlatform.ObjectModel/TestCase.cs

```diff
@@ -66,64 +63,6 @@ public TestCase(string fullyQualifiedName, Uri executorUri, string source)
             this.Source = source;
             this.LineNumber = -1;
         }
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="TestCase"/> class.
-        /// </summary>
-        /// <param name="fullyQualifiedName">
-        /// Fully qualified name of the test case.
-        /// </param>
-        /// <param name="method">
-        /// Managed method to extract the values of <see cref="TestCase.ManagedType"/> and <see cref="TestCase.ManagedMethod"/>.
-        /// </param>
-        /// <param name="executorUri">
-        /// The Uri of the executor to use for running this test.
-        /// </param>
-        /// <param name="source">
-        /// Test container source from which the test is discovered.
-        /// </param>
-        /// <remarks>
-        /// If <paramref name="method"/> is specified, TestId will be calculated based on that instead of <paramref name="fullyQualifiedName"/>.
-        /// </remarks>
-        public TestCase(string fullyQualifiedName, MethodBase method, Uri executorUri, string source)
```

Comment:
Isn't this breaking?

Reply:
This was introduced as part of FQN changes. The current release doesn't have these.